cmake/pcl_find_cuda.cmake

```diff
@@ -31,11 +31,11 @@ if(CUDA_FOUND)
   # https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html#deprecated-features
   
   if(NOT ${CUDA_VERSION_STRING} VERSION_LESS "11.0")
-    set(__cuda_arch_bin "5.2 5.3 6.0 6.1 7.0 7.2 7.5")
+    set(__cuda_arch_bin "3.5 3.7 5.0 5.2 5.3 6.0 6.1 6.2 7.0 7.2 7.5 8.0 8.6")
```

Comment:
> CUDA SDK 11.0 support for compute capability 3.5 - 8.0

You've added 8.6 to the list

Reply:
I would expect cuda 11 to support 8.6, which is the new RTX30xx cards that are coming out. But I guess we can wait until they come out to be sure.